wtp/wtpResultsParser.js

```diff
@@ -61,6 +58,15 @@ const parseTestResults = (testJson) => {
         userAgent = head.split(':').pop();
       }
     });
+    // make sure LCP image is in image list to analyze
+    const inList = imageList.findIndex((image) => {
+      return image.url === lcpURL;
+    });
+    if (inList === -1) {
+      imageList.push(origImageList.find((image) => {
```

Comment:
Wouldn’t normally origImageList will contain the imageList first? In that case there’s no real reason checking both right?

Reply:
origImageList is the list before it gets filtered there is a chance that the LCP would get filtered out that is why I check to make sure it is there